src/main/java/org/fundacion/jala/converter/controller/VideoConverterController.java

```diff
@@ -64,17 +64,16 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
                              final @RequestParam("height") int height,
                              final @RequestParam("audio") boolean audio,
                              final @RequestParam("checksum") String checksum,
-                             final @RequestParam("metadata") String metadata) throws IOException, InterruptedException {
-        paramChecksum = ChecksumFacade.getChecksum(checksum, file);
-        VideoParameter videoParam;
-        String path = paramChecksum.getOutputFilename();
-        videoParam = new VideoParameter(path, outputFormat, resolution, thumbnail, frameRate, width, height, audio);
-        String outputFilename = ConverterFacade.getVideoConverter(videoParam);
+                             final @RequestParam("metadata") boolean metadata)
+            throws IOException, InterruptedException {
+        LOGGER.info("start");
+        parameterOutputChecksum = ChecksumFacade.getChecksum(checksum, file);
+        String outputFilename = ConverterFacade.getVideoConverter(
+                new VideoParameter(parameterOutputChecksum.getOutputFilename(), outputFormat, resolution, thumbnail,
+                        frameRate, width, height, audio));

```

Comment:
These lines of code must be aligned and reach up to 120 characters

Reply:
I did, this configuration is given by intellij